src/components/TipList/TipList.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+//CSS
+import "./TipList.scss";
+
+import Tip from "../Tip/Tip.jsx";
```

Comment:
Utilizar imports con rutas absolutas y sin la extensión para componentes

```suggestion
import Tip from "components/Tip/Tip";
```